docs/setup/compile.md

```diff
@@ -94,7 +97,8 @@ pipenv --python 3.7
 5. Install the PySpark version and other dependency
 ```
 cd python
-pipenv install pyspark==3.0.1
+pipenv install pyspark
+pipenv install shapely~=1.7
```

Comment:
Why is Shapely needed here? Why not `pipenv install` since shapely is already listed here: https://github.com/apache/sedona/blob/master/python/Pipfile#L14

Reply:
Noted, removing this step.